app/__init__.py

```diff
@@ -328,6 +330,22 @@ def make_session_permanent():
     session.permanent = True
 
 
+def create_url(environment, url):
+    url_created = urlparse(url)
+    if environment == "production":
+        url_list = list(url_created)
+        url_list[1] = "beta.notify.gov"
+        url_for_redirect = urlunparse(url_list)
+        return url_for_redirect
+    return url
+
+
+def redirect_notify_to_beta():
+    url_to_beta = create_url(current_app.config["NOTIFY_ENVIRONMENT"], request.url)
+    if current_app.config["NOTIFY_ENVIRONMENT"] == "production":
```

Comment:
We could swap these two lines here so that the redirect only happens in the production environment:

```suggestion
    if current_app.config["NOTIFY_ENVIRONMENT"] == "production":
        url_to_beta = create_beta_url(request.url)
```

And in doing so, the `create_url` method would no longer need to check for which environment it's working with either.

Note that you'd also have to rename the method based on the suggestion above.